src/app/base/components/record-card/record-card-helper.ts

```diff
@@ -0,0 +1,29 @@
+import { DataBrowserModals, RecordCommentsModal } from "src/app/data/components/data-browser/helper-classes/modals-helper";
+import { SimilarSearch } from "src/app/data/components/data-browser/helper-classes/search-similar";
+import { ActiveSearchParams, AttributeSort, DataBrowserRecord } from "./record-card.types";
+
+export type RecordCardOptions = {
+    /**
+     * Options set for kern record card
+     * @attributesSortOrder {AttributeSort[] | any[], optional} - Array of sorted attributes to be displayed
+     * @dataBrowserModals {DataBrowserModals, optional} - Data browser modals with specific fields
+     * @indexExtendedRecord {number, optional} - Index of the extended record in the record list
+     * @recordList {DataBrowserRecord[], optional} - Array of records to be displayed
```

Comment:
this doesn't make sense to me, why does the record card (singular instance) need a reference to the whole list?
As discussed before we should look for the stuff we explicitly need and only use that. For this instance I think an output event for clicked index should do it (the stage of the record ids would then be handled by the databrowser though)